backend/frontend/templates/frontend/base.html

```diff
@@ -25,7 +25,17 @@
       crossorigin="anonymous"
     ></script>
 
-    
+    <style>
+      .htmx-indicator {
+        display: none;
+      }
+      .htmx-request .htmx-indicator{
+          display:inline-flex;
+      }
+      .htmx-request.htmx-indicator{
+          display:inline-flex;
+      }
+    </style>
```

Comment:
This looks a bit weird, is this not something we can add to the theme and style using Tailwind?

The last 2 selectors are almost similar and do the same thing maybe we just separate them with a comma?

Reply:
As per the docs [here](https://htmx.org/attributes/hx-indicator/#:~:text=When%20a%20request%20is%20in%20flight%2C%20this%20will%20cause%20the%20htmx%2Drequest%20class%20to%20be%20added%20to), htmx adds that class to the loading indicator and I could not find another way of making it work without making it messy. 

as for the second part about the "duplicated" selectors, I thought that it'd be easier to read that way as opposed to separating them with a comma, because then, it'd be harder to justify while reading that they aren't duplicates.